sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -196,7 +198,7 @@
   public static Read<Row> readRows() {
     // .withBeamSchema(true) means the internal implementation will prefer usage of JdbcIO.BeamSchemaRowMapper over JdbcIO.RowMapper for ResultSet mapping
     return JdbcIO.<Row>read()
-            .withBeamSchemaRowMapper(BeamSchemaRowMapperImpl.getInstance())
+            .withAutoSchemaSupport(true)
```

Comment:
withAutoSchemaSupport(boolean) is a private method so it is not exposed to the external user. It acts as a flag for internal implementation.